stacks-blockchain/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM blockstack/stacks-blockchain:2.0.5-stretch as build
+FROM blockstack/stacks-blockchain:fix-atlas-fixes-stretch as build
```

Comment:
Note: need to update this once stacks-blockchain version is released with atlas re-enabled